tests/console/oprofile.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple tests for oprofile.
+# - profile example application (fio to create some load)
+# - check report
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Install fio as load generator and oprofile
+    zypper_call "in fio oprofile";
+
+    # Start a system wide profiling
+    script_run "(operf --system-wide &)";
+
+    # Start load and stop profiler when finished
+    assert_script_run "fio --name=randwrite --filename=/tmp/randwrite --bs=4k --size=1G; killall -s SIGINT operf";
+
+    # Make sure the report is generated correctly and also check for an entry for the fio load
+    assert_script_run "opreport";
```

Comment:
You can maybe save the output of this to a file and upload it as log because it's too long now.